output/out_logzio.go

```diff
@@ -229,10 +231,22 @@ func initConfigParams(ctx unsafe.Pointer) error {
 	proxyUser := output.FLBPluginConfigKey(ctx, "proxy_user") // admin
 	proxyPass := output.FLBPluginConfigKey(ctx, "proxy_pass") // password1234
 
+	headers := make(map[string]string)
+	headerConfig := output.FLBPluginConfigKey(ctx, "headers")
+	if headerConfig != "" {
+		for _, header := range strings.Split(headerConfig, ",") {
```

Comment:
I think we need some validation for the header structure here for cases where the headers string is malformed like missing `:`  or for cases where we cant parse it correctly and add a warning log

Reply:
I updated the initConfigParams function to handle header configuration more robustly. The updated function now parses the headerConfig string, checking for correct key-value formatting. It logs warnings for malformed headers, headers without keys, and duplicate keys, where the latter will overwrite the previous value.